### ISSUE TITLE
Error when custom overlay return SVG element

### ISSUE THREAD

**Issue by iGarym** [jsplumb/jsplumb#794]
- jsPlumb: v2.8.2
- Browser: v70.0.3538.77

I am return a SVG element in the Custom Overlay create method, but it's crushed. Because the `className` property on SVG element have different definitions from the `className` property of other HTML element.

Crush on `defaults.js` [line-1244](https://github.com/jsplumb/jsplumb/blob/2e38e70506a2b401e67b01808546196b61494d28/src/defaults.js#L1244);

I am not sure how much it will be affected by changing it to a `classList` API, so no PR here.

**Comment by sporritt** [comment]
that code can probably use `jsPlumb.addClass` instead of a direct assignment to `className`, as SVG elements are handled by the `addClass` method.

So this line

```
 div.className = this._jsPlumb.instance.overlayClass + " " +
                    (this.cssClass ? this.cssClass :
                        params.cssClass ? params.cssClass : "");
```

could, i think, become

```
 jsPlumb.addClass(div, this._jsPlumb.instance.overlayClass + " " +
                    (this.cssClass ? this.cssClass :
                        params.cssClass ? params.cssClass : ""));
```

maybe you could try that with your overlay and see what happens.

**Comment by iGarym** [comment]
It works find when use `addClass`.
I have read the source code of `addClass` method. The methos handles some boundary conditions, such as `addClass(el, 'class1 class2')`, seems to be a complete replacement for `className`

**Comment by sporritt** [comment]
thanks for the update. i've made this change and released it in 2.8.4 today.
